step-templates/put-server-back-in-azure-load-balancer.json

```diff
@@ -0,0 +1,68 @@
+{
+  "Id": "540de6f6-f1c6-4e1c-9062-9e78570074f6",
+  "Name": "Put Server Back In Azure Load Balancer",
+  "Description": "Put Server Back In Azure Load Balancer",
```

Comment:
As this step template requires the use of a Service Principal, could you mention that in the description?